doc/source/whatsnew/v2.0.0.rst

```diff
@@ -291,6 +291,35 @@ The new behavior, as for datetime64, either gives exactly the requested dtype or
    ser.astype("timedelta64[s]")
    ser.astype("timedelta64[D]")
 
+.. _whatsnew_200.api_breaking.zero_len_indexes:
+
+Empty DataFrames/Series will now default to have a ``RangeIndex``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before, constructing an empty (length 0) :class:`Series` or :class:`DataFrame` without
```

Comment:
```suggestion
Before, constructing an empty (where ``data`` is ``None`` or an empty list-like argument) :class:`Series` or :class:`DataFrame` without
```

From your explanation in this PR

Reply:
👍

Reply:
Oh looks like the suggested change wasn't updated in the PR?

Reply:
Hmm, There was some conflict when I rebased, meybe I inadvertly took it out again. But it's back in now.